packages/slice-machine/src/features/settings/git/GitRepositoriesList.tsx

```diff
@@ -0,0 +1,76 @@
+import { Button, Skeleton, Text } from "@prismicio/editor-ui";
+import type { GitRepo, GitRepoSpecifier } from "@slicemachine/manager";
+import type { FC } from "react";
+
+import { FieldSetList, FieldSetListItem } from "@src/components/FieldSet";
+import { RelativeTime } from "@src/components/RelativeTime";
+import { GitRepositoryConnectDialog } from "@src/features/settings/git/GitRepositoryConnectDialog";
+import { GitRepositoryDisconnectDialog } from "@src/features/settings/git/GitRepositoryDisconnectDialog";
+import { useLinkedGitRepos } from "@src/features/settings/git/useLinkedGitRepos";
+
+type GitRepositoriesListProps =
+  | { mode: "link"; repos: GitRepo[] }
+  | { mode: "unlink"; repos: GitRepoSpecifier[] };
+
+export const GitRepositoriesList: FC<GitRepositoriesListProps> = ({
+  mode,
+  repos,
+}) => {
+  const { linkRepo, unlinkRepo } = useLinkedGitRepos();
+  return (
+    <FieldSetList>
+      {mode === "link"
+        ? repos.map((repo) => (
+            <FieldSetListItem
+              action={
+                <GitRepositoryConnectDialog
+                  linkRepo={linkRepo}
+                  repo={repo}
+                  trigger={<Button color="grey">Connect</Button>}
+                />
+              }
+              key={`${repo.provider}@${repo.id}`}
+            >
+              {repo.name}
```

Comment:
Is it possible to link the repo name to the GitHub repository? It would give users a quick way to open their repository and check its status.

Reply:
You're suggesting adding a way to open the GitHub repository from the setting page?

Something like this? 

![image](https://github.com/prismicio/slice-machine/assets/6057521/5b4c12c3-2ed9-4307-8d8c-82650bd18b09)

Reply:
That's less obvious, but it works.

Reply:
Let's do the suggestion then.